ledger/src/test_helpers/chain_builder.rs

```diff
@@ -127,19 +152,27 @@ impl<N: Network> TestChainBuilder<N> {
     }
 
     /// Initialize the builder with the default quorum size.
-    pub fn new(rng: &mut TestRng) -> Result<Self> {
-        Self::new_with_quorum_size(4, rng)
+    pub fn new(rng: &mut TestRng, transactions: Vec<Transaction<N>>) -> Result<Self> {
```

Comment:
It might be nicer to have this as a separate constructor and default to no transactions.